snakemake/workflow.py

```diff
@@ -243,7 +250,13 @@ def norm_rule_relpath(f, rule):
         for f in self.included:
             f = local_path(f)
             if f:
-                files.add(os.path.relpath(f))
+                if not (
+                    ON_WINDOWS
+                    and os.path.splitdrive(f)[0] != os.path.splitdrive(os.getcwd())[0]
+                ):
+                    # relpath doesn't work on win if files are on different drive
+                    f = os.path.relpath(f)
+                files.add(f)
```

Comment:
@johanneskoester Will this work instead? 
```suggestion
                try:
                    f = os.path.relpath(f)
                except ValueError:
                    # relpath raise ValueError on Win if files are on different drive, we ignore it
                    if not ON_WINDOWS:
                        raise
                files.add(f)
```

Reply:
Nice!